stable/test/integration/_exec.pony

```diff
@@ -0,0 +1,82 @@
+use "ponytest"
+use "files"
+use "process"
+use "regex"
+
+actor _Exec
+  new create(
+    h: TestHelper,
+    cmdline: String,
+    notifier: ProcessNotify iso)
+  =>
+    try
+      let args = cmdline.split_by(" ")
+      let path = FilePath(h.env.root as AmbientAuth, "build/release/stable")?
+      let vars: Array[String] iso = []
+      let auth = h.env.root as AmbientAuth
+      let pm: ProcessMonitor = ProcessMonitor(auth, auth, consume notifier,
+        path, consume args, consume vars)
+      pm.done_writing()
+    else
+      h.fail("Could not create FilePath!")
+    end
+
+class _ExpectClient is ProcessNotify
+  let _h: TestHelper
+  let _out: Array[String] val
+  let _err: Array[String] val
+  let _code: I32
+  var _status: Bool
+
+  new iso create(
+    h: TestHelper,
+    stdout': (Array[String] val | None),
+    stderr': (Array[String] val | None),
+    code': I32)
+  =>
+    _h = h
+    _out = match stdout'
+      | None => recover Array[String](0) end
+      | let a: Array[String] val => a
+      end
```

Comment:
The style here doesn't quite match the official standard library style guide. See [this rule](https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#assignment) specifically, indicating that the `match` keyword should be on the next line and indented.

Also, I think this could be a bit more succint. The `recover Array[String](0) end` could be just `[]` and let the compiler infer the type.